apps/api/src/view-models/buildViewModelForResource/viewModels/utilities/graph/buildTreeFromNodes.spec.ts

```diff
@@ -0,0 +1,14 @@
+import buildTestData from '../../../../../test-data/buildTestData';
+import buildTreeFromNodes from './buildTreeFromNodes';
+
+describe('buildTreeFromNodes', () => {
```

Comment:
Smallest test ever!

One question is whether we should hard-wire a set of category data here, so we don't have to continuously validate the snapshot every time we tweak the test data. This might lower the value of this test due to poor maintainability.

Reply:
I'll defer to you on this.